### DISCUSSION TITLE
Access to v0.17.0 of this library?

### DISCUSSION THREAD

**Discussion by teevertech** [hathach/tinyusb#2701] [Q&A]
I am interested some of the features that are available in the 0.17.0 version of this library

The specific feature that I'm looking for are the support for STM32U5 microcontrollers and added support for multiple interfaces in UVC.

Is there any wayt to access this WIP version, or is there a time line on when it will be released?

**Comment by HiFiPhile** [comment]
It's not released yet, just click "Download ZIP" on home page.

Or https://github.com/hathach/tinyusb/archive/refs/heads/master.zip
